### PR TITLE
Add mysql 5.7 and 8.x to builder image

### PR DESCRIPTION
**Pull request by tlbdk**

### CODE REVIEW

**Comment by tlbdk** [comment]
https://app.shortcut.com/connectedcars/story/81671/add-mysql-5-7-x-and-8-x-to-docker-builder

**Review by chribsen — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -141,22 +141,41 @@ RUN apt-get update -qq && \
...
 
+# Make sure we use mysql-server from Ubuntu 18.04 as this is the last version with mysql 5.7
 RUN if [ "$TARGETOS/${TARGETARCH}" = "linux/amd64" ]; then \
 		echo Downloading amd64 binaies; \
 		add-apt-repository "deb http://mirrors.kernel.org/ubuntu/ bionic main"; \
+		add-apt-repository "deb http://mirrors.kernel.org/ubuntu/ bionic-security main"; \
 	elif [ "$TARGETOS/${TARGETARCH}" = "linux/arm64" ]; then \
 		echo Downloading arm64 binaies; \
 		add-apt-repository "deb http://ports.ubuntu.com/ubuntu-ports bionic main"; \
+		add-apt-repository "deb http://ports.ubuntu.com/ubuntu-ports bionic-security main"; \
```
Comment (LuisFros) [inline Dockerfile:152]: 💯

File: test/test.js [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+    let mySqlClient
+    try {
+      mySqlClient = new MySQLClient({ port: await mySqlServer.getListenPort() })
+      const pool = await mySqlClient.getConnectionPool('mysql')
+      const databases = (await mySqlClient.query(pool,
+        `
+          SHOW VARIABLES LIKE 'version';
+        `
+      )).map(r => `${r.Value}`)
+      expect(databases).to.match(/^(5\.7|8\.)/)
```
Comment (LuisFros) [inline test/test.js:22]: Do you think it would be good to test that `5.7.21` is no longer a version? Considering "breaking changes"

Comment (LuisFros) [reply test/test.js:22]:
```suggestion
      expect(databases).to.match(/^(5\.7|8\.)/)
      expect(databases.indexOf('5.7.21')).to.equal(-1)
```

**Review by LuisFros — APPROVED** [review]

**Pull request merged**